core/math/big/prime.odin

```diff
@@ -44,7 +44,7 @@ internal_int_prime_is_divisible :: proc(a: ^Int, allocator := context.allocator)
 	Computes res == G**X mod P.
 	Assumes `res`, `G`, `X` and `P` to not be `nil` and for `G`, `X` and `P` to have been initialized.
 */
-internal_int_exponent_mod :: proc(res, G, X, P: ^Int, allocator := context.allocator) -> (err: Error) {
+internal_int_exponent_modulus :: proc(res, G, X, P: ^Int, allocator := context.allocator) -> (err: Error) {
```

Comment:
In most of the package, the procedures are named `*_modulo`, it's the parameter that's called `modulus`.
![image](https://github.com/odin-lang/Odin/assets/194614/efa7233e-b8cc-4a2c-af2c-3c2ddf108648)

Reply:
You are correct. Let me fix that.